### ISSUE TITLE
Different Trial Output Format

### ISSUE THREAD

**Issue by srilman** [keras-team/keras-tuner#259]
Hi. First of all, thank you for providing such a wonderful library! I was really happy to move away from my manual Tensorflow Hyperparameter Searching solution to this library. I'd love to see this project grow even further.

Sorry if this sounds picky, but I really am not a huge fan of the printed output, especially for the functions `tuner.search_space_summary()` or `tuner.search()`. First of all, the text are sometimes hard to read, especially on Google Colab in Dark Mode, because they are specially formatted and colored oddly. Here is an example of what I mean:
![image](https://user-images.githubusercontent.com/17444967/77282923-8da29b80-6ca1-11ea-892e-fd663656ae3e.png)

Secondly, the output for search is usually pretty long, but it's all printed vertically and contains a lot of redundant and verbose info; it becomes difficult to parse out the important pieces of info. While this is not an immediate annoyance, it does get frustrating to use over time.

I would like to propose that 
1) The normal standard out font is used instead of what is currently used.
2) Output is formatted in a tabular manner, with columns for the different values. There are many libraries that can format tabular data and maybe some that can mimic the formatting of Javascript's `console.table` which outputs some really pretty tables (but might not work on all screens).

I'd be happy to contribute with a PR since this doesn't seem especially difficult to do. Thanks!

**Comment by ben-arnao** [comment]
I Agree. The standard output is not very useful IMO. In my fork the metrics i use are the score of the trials, amount of configs tested, len of search space, epochs of the trial, and time elapsed of the trial. I've even included a linear regression of trial scores to see if the tuner is indeed finding better configs. 

Either way i definitely think for most use cases the amount of parameters you'll be tuning is too much to fit on the screen. Better to just give a trial summary and summary of the tuner up until this point.

**Comment by Aviously** [comment]
Hi @srilman @ben-arnao, this had been updated in a recent PR (https://github.com/keras-team/keras-tuner/pull/324). The output for hyperparameters during a search is displayed using standard `print()` and also shows more relevant information.

```
Trial 3 Complete [00h 00m 01s]
val_accuracy: 0.9438999891281128

Best val_accuracy So Far: 0.9476000070571899
Total elapsed time: 00h 00m 10s

Search: Running Trial #4

Hyperparameter    |Value             |Best Value So Far
nodes             |110               |140
nodes_2           |130               |190
```

**Comment by haifeng-jin** [comment]
I assume this is resolved with the latest release of keras-tuner 1.0.2.
I am closing this issue.
Please let me know if it doesn't solve the issue.
